src/components/views/settings/BridgeTile.js

```diff
@@ -23,7 +23,8 @@ import Pill from "../elements/Pill";
 import {makeUserPermalink} from "../../../utils/permalinks/Permalinks";
 import BaseAvatar from "../avatars/BaseAvatar";
 import AccessibleButton from "../elements/AccessibleButton";
-
+import {replaceableComponent} from "../../../utils/replaceableComponent";
```

Comment:
somehow you've destroyed the blank line between this and the decorator - plz fix

Reply:
:boom:

Reply:
![image](https://user-images.githubusercontent.com/2072976/73260550-0580a800-41c2-11ea-97a6-6366840826f2.png)

I suspect I mistook this for a blank line.